okhttp-urlconnection/README.md

```diff
@@ -1,10 +0,0 @@
-OkHttp URLConnection
```

Comment:
Could you revert these deletes? I’d like to continue to build & ship this old artifact forever. Breaking users sucks.